scripts/github-actions/triage/comment_workflow.js

```diff
@@ -0,0 +1,187 @@
+                }
+              }
+              projectV2(number: $project) {
+                id
+                field(name: "Status") {
+                  ... on ProjectV2SingleSelectField {
+                    id
+                    options {
+                      id
+                      name
+                    }
+                  }
+                }
+              }
+            }
+          }`;
+        
+        const getItemInfoVars = {
+            org: context.payload.organization.login,
+            repo: context.payload.repository.name,
+            issue: issueOrPullRequest.number,
+            project: 9
+        };
+
+        const getItemInfo = await github.graphql(getItemInfoQuery,getItemInfoVars);
+
+        const issueDataFromGraphQL = getItemInfo.organization.repository[issueOrPullRequest.type];
+        const projectID = getItemInfo.organization.projectV2.id;
+        //using let because we will need to override this value once we add the item to the project board if it isn't already on it
+        let projectItemID = issueDataFromGraphQL.projectItems.nodes.length > 0 ? issueDataFromGraphQL.projectItems.nodes[0].id : null;
+        const isItemArchived = issueDataFromGraphQL.projectItems.nodes.length > 0 ? issueDataFromGraphQL.projectItems.nodes[0].isArchived : false;
+        const statusFieldID = getItemInfo.organization.projectV2.field.id;
+        const status = "New Issue"; // You can hardcode this value, or extract it from the JSON object if needed
+        const newStatusColumnID = getItemInfo.organization.projectV2.field.options.find(option => option.name === "New Issue").id;
+       
+        // If issue is archived on the board, reactivate it
+        if(isItemArchived) {
+          const unarchiveQuery = `
+          mutation ($project_id: ID!, $item_id: ID!) {
+            unarchiveProjectV2Item(input: {projectId: $project_id, itemId: $item_id}) {
+              clientMutationId
+              item {
+                id
+              }
+            }
+          }`;
+
+          const unarchiveQueryVars = {
+            project_id: projectID,
+            item_id: projectItemID,
+          };
+
+          const unarchiveItem = await github.graphql(unarchiveQuery,unarchiveQueryVars); 
+          
+        };
+
+        // If the issue is not on the project board add it to the project board
+        if(projectItemID == null) {
+          const addToProjectBoardQuery = `
+            mutation ($project_id: ID!, $item_id: ID!) {
+              addProjectV2ItemById(input: {contentId: $item_id, projectId: $project_id}) {
+                clientMutationId
+                item {
+                  id
+                }
+              }
+            }`;
+
+          const addToProjectBoardQueryVars = {
+            project_id: projectID,
+            item_id: issueDataFromGraphQL.id,
+          };
+
+          const addToProjectBoard = await github.graphql(addToProjectBoardQuery,addToProjectBoardQueryVars);
```

Comment:
Can the `/.github/workflows/triage_add_to_project.yml` workflow be removed as well since script handles adding it?

Reply:
No that workflow is used by other repos to add tickets to the triage board

Reply:
is the though those repos will be updated to use this workflow and once updated, we can remove the add_to_project workflow? Just wondering if we will try to add it 2x with this update

Reply:
this is when comments are created.  The other actions are when issues are opened.  Different triggering events.